Bug: 1883036 - Option to not redeploy the service signer (#25976)
openshift/openshift-docs: install/configuring_inventory_file.adoc

contains the full chain of intermediate and root certificates for the CA.
See xref:../install_config/redeploying_certificates.adoc#redeploying-new-custom-ca[Redeploying a New or Custom {product-title} CA]

|`openshift_redeploy_service_signer`

> | If the parameter is set to `false`, the service signer will not be redeployed during `openshift-master/openshift-redeploy-ca.yml`. 

Reviewer: ```suggestion
| If the parameter is set to `false`, the service signer is not redeployed when you run the  `openshift-master/redeploy-certificates.yml` playbook.
```

> By default the variable is set to `true`.

Reviewer: ```suggestion
The default value is `true`.
```
